ccm_web/client/src/App.tsx

```diff
@@ -84,7 +107,7 @@ function App (props: AppProps): JSX.Element {
             </Route>
           </div>
           <Switch>
-            <Route exact={true} path="/" render={() => (<Home globals={globals} ltiKey={props.ltiKey} />)} />
+            <Route exact={true} path="/" render={() => (<Home globals={globals} ltiKey={props.ltiKey} course={course} setCourse={setCourse} />)} />
```

Comment:
Can you please explain how is `setCourse` working here? I never passed around a useState set unction as another component. Your approach is working as expected with out having to make an getCourse API call. I couldn't wrap my head around it so I am asking here.

Reply:
doLoadCourse gets called when globals are loaded, and then when the course name is changed from within the home component, it uses the setCourse handle passed in to update the course value in App.  The new value is coming from the value returned by the call to  set the course name.

Reply:
In regards to the above screenshot about the color difference in the course name title with and without edit roles, the color of text should be reversed. If a user is unable to edit, the text should be displayed in gray (indicates to users this cannot be changed). If a user can edit, the text would be black.

Reply:
Thanks @janelilr. I will create a new issue with your suggestions.